public/api/v2/config

```diff
@@ -13,6 +13,7 @@
     "email": true,
     "openid": true
   },
+  "bannerText": "Dies ist ein Testbanner",
```

Comment:
This should match the banner note and therefore be English.

Reply:
As this is just a mock call not really necessary to change, but technically correct

Reply:
While I agree that it's not necessary it would be great when mock data would be consistent.